app/views/includes/paye/paye_taxs_header_nav.scala.html

```diff
@@ -22,7 +22,7 @@
 
 <div class="header-proposition">
     <div class="content">
-        <nav id="proposition-menu" class="header__menu" role="navigation">
+        <nav id="proposition-menu" class="header__menu">
```

Comment:
If this is not the only nav element on the page you should look at giving it an aria label